src/main/java/io/reactivex/internal/schedulers/ExecutorScheduler.java

```diff
@@ -330,6 +331,11 @@ public void dispose() {
                 direct.dispose();
             }
         }
+
+        @Override
+        public Runnable getWrappedRunnable() {
+            return get();
```

Comment:
`dispose()` will swap in a null but the method is defined as `@NotNull`. So either the interface should allow nulls or this should return Functions.EMPTY_RUNNABLE:

```java
Runnable r = get();
return r != null ? r : Functions.EMPTY_RUNNABLE;
```

Reply:
Ehhh I did not notice it. I will check what will happen when dispose return `Functions.EMPTY_RUNNABLE`

Reply:
All tests were passed with this change. I will leave with:
```
Runnable r = get();
return r != null ? r : Functions.EMPTY_RUNNABLE;
```